setup.py

```diff
@@ -265,6 +265,10 @@ def cython_extensions_and_necessity():
 setuptools.setup(
   name='grpcio',
   version=grpc_version.VERSION,
+  description='HTTP/2-based RPC framework',
+  author='Google',
+  author_email='grpc-io@googlegroups.com',
+  url='http://www.grpc.io/docs/quickstart/python.html',
```

Comment:
Let's just go with `http://grpc.io` rather than the deep link directly to the quickstart. Unless this is supposed to be something like a tutorial or "getting started" URL?